docs/CODE_GUIDELINES.md

```diff
@@ -496,14 +552,34 @@ public:
 }
 ```
 
-### 8.6. Methods
+### 8.6. Structs
+Use PascalCase.
+```cpp
+struct InfoChar
+{
+  bool m_isBold{false};
+}
+```
+
+### 8.7. Methods
 Use PascalCase always, uppercasing the first letter even if the methods are private or protected.
+The method parameters start with lower case and follow CamelCase style, without type prefixing (Systems Hungarian notation).
```

Comment:
_("The" isn't needed here if you're talking about method parameters in general, e.g. "Roses are red"; it would be, however, if you're talking about specific method parameters, e.g. "The roses in the garden are red")_

Method parameters start with lower case and follow CamelCase style...